### PR TITLE
[Keyvault] Updating READMEs

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

File: sdk/keyvault/azkeys/README.md [hunk 1]
```diff
@@ -220,14 +237,14 @@ func ExampleClient_UpdateKeyProperties() {
 		Tags: map[string]string{
 			"Tag1": "val1",
 		},
-		KeyAttributes: &azkeys.KeyAttributes{
-			RecoveryLevel: azkeys.CustomizedRecoverablePurgeable.ToPtr(),
+		Properties: &azkeys.Properties{
+			RecoveryLevel: azkeys.DeletionRecoveryLevelCustomizedRecoverablePurgeable.ToPtr(),
```
Comment (kimprice) [inline sdk/keyvault/azkeys/README.md:241]: Wow, this is really long. Is this change to create a standard for enums to have the enum type prefix the name?

File: sdk/keyvault/azsecrets/README.md [hunk 2]
```diff
@@ -103,9 +103,30 @@ Once the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** en
...
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		panic(err)
+	}
+
+	client, err := azsecrets.NewClient(vaultURL, cred, nil)
+	if err != nil {
+		panic(err)
+	}
+	_ = client
```
Comment (kimprice) [inline sdk/keyvault/azsecrets/README.md:128]: Is something missing here?

Comment (seankane-msft) [reply sdk/keyvault/azsecrets/README.md:128]: That's a copy error, removing it now.

File: sdk/keyvault/azsecrets/README.md [hunk 3]
```diff
@@ -126,90 +147,189 @@ This section contains code snippets covering common tasks:
...
+
+	client, err := azsecrets.NewClient(vaultURL, cred, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	secret := "mySecret"
+	value := "mySecretValue"
+
+	resp, err := client.SetSecret(context.TODO(), secret, value, nil)
```
Comment (kimprice) [inline sdk/keyvault/azsecrets/README.md:174]:
```suggestion
	secretName := "mySecretName"
	secretValue := "mySecretValue"

	resp, err := client.SetSecret(context.TODO(), secretName, secretValue, nil)
```
It's clearer to specify the way you did previously with name and value.

**Review by RickWinter — APPROVED** [review]

**Pull request merged**